package.json

```diff
@@ -1,15 +1,22 @@
 {
   "name": "salesforce-jwt-promise",
-  "version": "0.1.8",
+  "version": "0.2.0",
```

Comment:
this is a minor bump.  We'd need a major bump since this isn't backwards compatible. 

https://semver.org/